scipy/stats/_continuous_distns.py

```diff
@@ -5800,6 +5800,12 @@ def _stats(self, nu):
         g2 /= nu*mu2**2.0
         return mu, mu2, g1, g2
 
+    def _fitstart(self, data, args=None):
+        # Empirically good first estimates
+        loc = data.mean()
+        scale = data.var()
+        return loc, scale
```

Comment:
```suggestion
        return loc, scale
```
`_fitstart` needs to return an estimate of the shape parameter(s), too.

Reply:
I forgot to commit all my changes before pushing at first. This is addressed in 1a0164d1c793d1670873f21472b811625cf7e04f.